Dockerfile

```diff
@@ -4,9 +4,14 @@ MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
 # Install nodejs dependencies
 COPY package.json /app/package.json
 WORKDIR /app
-RUN apt-get-install.sh npm && \
-    npm install --production && \
-    apt-get-purge.sh npm
+RUN apt-get-install.sh curl
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x -o nodesource_setup.sh && \
+	    bash nodesource_setup.sh && \
+	    apt-get-install.sh nodejs
+
+RUN npm install --production && \
```

Comment:
Add this to the previous RUN, otherwise it will create a new layer resulting in a bigger image.
So what you want is to have all these in 1 RUN command instead of 3 commands. This is specifically because you're doing a purge to cleanup space.